### PR TITLE
[OSDOCS-3872] 4.11 release notes for ShiftStack

### PR DESCRIPTION
**Pull request by maxwelldb**
Version(s): 4.11

Issue: [OSDOCS-3872](https://issues.redhat.com//browse/OSDOCS-3872)

Link to docs preview: http://file.rdu.redhat.com/~mbridges/411rns/release_notes/ocp-4-11-release-notes.html#ocp-4-11-networking

_Last generated:_ Wed Jul 27 05:10:11 PM UTC 2022

### CODE REVIEW

**Comment by maxwelldb** [comment]
Also relevant: #46521

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -390,6 +390,24 @@ Be aware that CIDR ranges for networks are not adjustable after cluster installa
...
 
+[id="ocp-4-11-openshift-on-openstack-hw-offloading"]
+==== OVS hardware offloading on {rh-openstack}
+
+For clusters that run on {rh-openstack}, link:https://www.openvswitch.org/[Open vSwitch (OVS)] hardware offloading is now generally available.
+
+For more information, see xref:../post_installation_configuration/network-configuration.adoc#nw-osp-enabling-ovs-offload_post-install-network-configuration[Enabling OVS hardware offloading].
+
+[id="ocp-4-11-openshift-on-openstack-nfv-experience"]
+==== OVS hardware offloading on {rh-openstack}
```
Comment (EmilienM) [inline release_notes/ocp-4-11-release-notes.adoc:401]: Probably a wrong copy/paste but you want to write NFV user-experience probably?

**Review by mandre — COMMENTED** [review]
Looks good, thanks @maxwelldb.

**Review by jldohmann — APPROVED** [review]
/lgtm

**Pull request merged**